src/python/pants/backend/python/rules/prepare_chrooted_python_sources.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from dataclasses import dataclass
+
+from pants.backend.python.rules.inject_init import InjectedInitDigest
+from pants.engine.fs import Digest, DirectoriesToMerge
+from pants.engine.legacy.graph import HydratedTarget, HydratedTargets
+from pants.engine.rules import rule
+from pants.engine.selectors import Get, MultiGet
+from pants.rules.core.strip_source_root import SourceRootStrippedSources
+
+
+@dataclass(frozen=True)
+class ChrootedPythonSourcesRequest:
+  hydrated_targets: HydratedTargets
+
+
+@rule
+async def prepare_chrooted_python_sources(request: ChrootedPythonSourcesRequest) -> Digest:
+  source_root_stripped_sources = await MultiGet(
+    Get[SourceRootStrippedSources](HydratedTarget, target_adaptor)
```

Comment:
`target_adaptor` is a misnomer (I know it was copied from before). Should be `hydrated_target`